lib/presentation/widgets/about_us_header.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:sis_group6/bloc/tweets/tweets_bloc.dart';
+import 'package:sis_group6/controller/menu_controller.dart';
+import 'package:sis_group6/core/utils/responsive.dart';
+// import 'package:sis_group6/presentation/widgets/search_bar.dart';
+
+class About_Us_Header extends StatelessWidget {
+  const About_Us_Header({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final bloc = BlocProvider.of<TweetsBloc>(context);
+    return Row(
+      children: [
+        if (!Responsive.isDesktop(context))
+          IconButton(
+            icon: const Icon(Icons.menu),
+            onPressed: context.read<MenuController>().controlMenu,
+          ),
+        if (!Responsive.isMobile(context))
+          Text(
+            "About Us",
```

Comment:
To make `Header` more generic, extract the content from this `Text` widget into a `final headerTitle` var and add it to the constructor

Reply:
done?